Tests/AEPOptimizeTests/IntegrationTests/OptimizeIntegrationTests.swift

```diff
@@ -105,7 +165,10 @@ class OptimizeIntegrationTests: XCTestCase {
                                           placementId: "xcore:offer-placement:1111111111111111")
         
         // update propositions
-        Optimize.updatePropositions(for: [decisionScope], withXdm: nil)
+        Optimize.updatePropositions(for: [decisionScope], withXdm: nil){ (scope,error) in
+            XCTAssertNotNil(scope)
```

Comment:
nit: Can you also add `XCTAssertNil(error)` for success case?

Reply:
Done